### ISSUE TITLE
Fix autodoc issues wrt mesh package

### ISSUE THREAD

**Issue by KineticTheory** [lanl/Draco#524]
Issues:

```
  F:/work/draco/src/mesh/Draco_Mesh.hh:161: warning: The following parameters of rtt_mesh::Draco_Mesh::compute_cell_to_cell_linkage(const std::vector< unsigned > &cell_type, const std::vector< unsigned > &cell_to_node_linkage, const std::vector< unsigned > &side_node_count, const std::vector< unsigned > &side_to_node_linkage, const std::vector< unsigned > &ghost_cell_type, const std::vector< unsigned > &ghost_cell_to_node_linkage) are not documented:
    parameter 'ghost_cell_type'
    parameter 'ghost_cell_to_node_linkage'
  F:/work/draco/src/mesh/X3D_Draco_Mesh_Reader.hh:148: warning: The following parameters of rtt_mesh::X3D_Draco_Mesh_Reader::map_x3d_block(const std::string &block_name, size_t &dist) are not documented:
    parameter 'dist'
```

**Comment by KineticTheory** [comment]
This isn't super high priority. Maybe address this in your next code change to the `mesh` package.

**Comment by RyanWollaeger** [comment]
@KineticTheory Thank you for catching this - it will be fixed in the next changeset.  It also may be possible to remove some of the arguments in the cell_to_cell linkage function.

**Comment by KineticTheory** [comment]
Fixed by #550
